ui/balances_table.ts

```diff
@@ -43,18 +43,36 @@ function create_full_data(): AssetBalance[] {
         if (!inter_data.hasOwnProperty(asset)) {
             continue;
         }
-        total_usd += inter_data[asset]['usd_value'] as number;
+        let asset_usd_value;
+        if (typeof inter_data[asset].usd_value === 'string') {
```

Comment:
@kelsos is there any other way in typescript to do this? Feels not so clean.

Reply:
I tried the classic JavaScript approach of multiplying by 1 but it doesn't seem to work with the TypeScript type system.

Reply:
For the Vue rewrite the parsing is happening on the edges so there will not be such a problem

Reply:
that's great. I actually can't wait :)